src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -51,45 +63,273 @@ function determinePillColour(status: ActivityStatus, type: ActivityType) {
   display: flex;
+  flex-flow: column wrap;
+  flex: 0 1 auto;
+  justify-content: center;
+`
 
-  & > a {
+const StatusLabel = styled.div<{ isPending: boolean }>`
+  height: 28px;
+  width: 100px;
+  border: ${({ isPending, theme }) => isPending && `1px solid ${theme.border2}`};
+  color: ${({ color }) => color};
+  position: relative;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: 600;
+
+  &::before {
+    content: '';
+    background: ${({ color, isPending }) => (isPending ? 'transparent' : color)};
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100%;
     width: 100%;
+    border-radius: 4px;
+    opacity: 0.1;
+  }
+
+  > svg {
+    margin: 0 5px 0 0;
   }
 `
 
+const TransactionWrapper = styled.div`
+  width: 100%;
+  border-radius: 0;
+  font-size: initial;
+  display: flex;
+  margin: 0;
+  padding: 16px;
+  transition: background 0.2s ease-in-out;
+
+  &:hover {
+    background: rgba(217, 232, 239, 0.35);
+  }
+`
+
+const StatusLabelBelow = styled.div<{ isCancelling?: boolean }>`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 12px;
+  line-height: 1.1;
+  margin: 7px auto 0;
+  color: ${({ isCancelling, theme }) => (isCancelling ? theme.primary1 : 'inherit')};
+
+  ${LinkStyledButton} {
+    margin: 2px 0;
+  }
+`
+
+function getActivitySummary(params: { id: string; activityData: ReturnType<typeof useActivityDescriptors> }) {
+  const { id, activityData } = params
+
+  if (!activityData) return null
+
+  const { activity, type, summary } = activityData
+
+  // TODO: probably get rid of this?
+  const orderSummary: { from: string | undefined; to: string | undefined; validTo: string } = {
+    from: '🤔',
+    to: '🤔',
+    validTo: '🤔',
+  }
+  const isOrder = type === ActivityType.ORDER
+
+  if (isOrder) {
+    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo } = activity as Order
+
+    const sellAmt = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString())
+    const feeAmt = CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
+    const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+
+    orderSummary.from = formatSmart(sellAmt.add(feeAmt))
+    orderSummary.to = formatSmart(outputAmount)
+    orderSummary.validTo = new Date((validTo as number) * 1000).toLocaleString()
+  }
+
+  return (
+    <Summary>
+      <b>{type} ↗</b>
+      <div>
+        {isOrder ? (
+          <>
+            <span>
+              <b>From</b>
+              <i>{orderSummary.from}</i>
+            </span>
+            <span>
+              <b>To at least</b>
+              <i>{orderSummary.to}</i>
+            </span>
+            <span>
+              <b>Valid to: {orderSummary.validTo}</b>
+            </span>
+          </>
+        ) : (
+          summary ?? id
+        )}
+      </div>
+    </Summary>
+  )
+}
+
```

Comment:
@biocom updated
![screenshot_2021-07-26_16-08-22](https://user-images.githubusercontent.com/43217/127070528-bcc955b5-b51b-4f04-bb68-d1cfb39e4d09.png)

Now I'm thinking, this function could as well be in a utils file or something.
We'll want to reuse that for updating the toast notifications as well.
Although that should be a different task/PR.

I also merged it with latest `develop`